bgpd/bgp_route.h

```diff
@@ -71,7 +71,8 @@ enum bgp_show_adj_route_type {
 	"               i internal, r RIB-failure, S Stale, R Removed\n"
 #define BGP_SHOW_OCODE_HEADER "Origin codes:  i - IGP, e - EGP, ? - incomplete\n\n"
 #define BGP_SHOW_NCODE_HEADER "Nexthop codes: @NNN nexthop's vrf id, < announce-nh-self\n"
-#define BGP_SHOW_HEADER "   Network          Next Hop            Metric LocPrf Weight Path\n"
+#define BGP_SHOW_HEADER "   Network             Next Hop         Metric LocPrf Weight Path\n"
+#define BGP_SHOW_HEADER_WIDE "   Network                                      Next Hop                                  Metric LocPrf Weight Path\n"
```

Comment:
Could you test this wide option when `bgp default show-nexthop-hostname` is enabled? And show the output here?

Reply:
![sh_nexthop_hostname](https://user-images.githubusercontent.com/10223016/87692867-e0044c80-c7a9-11ea-905a-184f48deb62b.png)

Reply:
somehow misaligned :/

Reply:
You mean the Metric part ?

Reply:
metrics and the rest

Reply:
please check the output with default show-nexthop-hostname set
![sh_adv_routes](https://user-images.githubusercontent.com/10223016/87825623-166ac600-c895-11ea-9354-dea304164138.png)
![sh_bgp](https://user-images.githubusercontent.com/10223016/87825630-18348980-c895-11ea-8212-c10387930267.png)
![sh_ip_bgp](https://user-images.githubusercontent.com/10223016/87825635-1965b680-c895-11ea-9f16-e143373af212.png)
![sh_ip6_bgp_unicast](https://user-images.githubusercontent.com/10223016/87825639-19fe4d00-c895-11ea-991f-b82330f23f6d.png)
![sh_received_routes](https://user-images.githubusercontent.com/10223016/87825646-1a96e380-c895-11ea-9ba8-88d79a7f5f76.png)

Reply:
looks good now